src/main/kotlin/no/nav/syfo/cronjob/CronjobModule.kt

```diff
@@ -110,4 +141,12 @@ fun Application.cronjobModule(
     ) {
         cronjobRunner.start(cronjob = journalpostDistribusjonCronjob)
     }
+
+    if (environment.resendNarmesteLederVarselEnabled) {
```

Comment:
Nå tenker jeg høyt her: Sånn jeg skjønner det, ønsker vi å kjøre jobbe bare én gang slik at ikke varslene sendes flere ganger. Tenker vi å sette toggelen til false som neste commit til master, for å unngå at denne jobben kjører flere ganger?
Og hvor mange podder er det som kjører? Er det noe mekanikk som gjør at denne jobben kun vil kjøre på en av poddene?

Reply:
Ja, nå har jeg bare satt 9999999 minutter delay mellom kjøringer, så tenker jeg at vi rekker å disable før den kjører på nytt 😅 Det er en sjekk på at kun poden som er `leader` kjører cronjobber 👍